Bottom: 0184b89c903695ef19182b6d6bc473de0aaefda4
Top:    98dc07f42f39d83f35d6a7ccd26824ca2b5f9583
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-02-07 20:54:07 +0800

Refresh of vhost-packed-ring-support

---

diff --git a/drivers/vhost/vhost.h b/drivers/vhost/vhost.h
index 7876a3d..519d9df 100644
--- a/drivers/vhost/vhost.h
+++ b/drivers/vhost/vhost.h
@@ -91,7 +91,10 @@ struct vhost_virtqueue {
 	/* The actual ring of buffers. */
 	struct mutex mutex;
 	unsigned int num;
-	struct vring_desc __user *desc;
+	union {
+		struct vring_desc __user *desc;
+		struct vring_packed __user *packed;
+	};
 	struct vring_avail __user *avail;
 	struct vring_used __user *used;
 	const struct vhost_umem_node *meta_iotlb[VHOST_NUM_ADDRS];
